docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '2.1'
+
+services:
+  php-server:
```

Comment:
You should add the environment variables so that the server would connect to mongodb when ran with docker-compose, same for the docker-compose.debug.yml